pkg/container-hook/runtime-finder/finder.go

```diff
@@ -43,6 +43,7 @@ var RuntimePaths = []string{
 	"/var/lib/rancher/k3s/data/current/bin/runc", // Used in k3s
 	"/var/lib/rancher/rke2/bin/runc",             // Used in RKE2
 	"/usr/libexec/crio/runc",                     // Used in kubeadm on Debian, upstream crio
+	"/var/lib/k0s/bin/runc",                      // Used in k0s
```

Comment:
The change looks good to me.

Is it from https://k0sproject.io/? I didn't know the project.

Could you rebase your branch and make it a single commit instead of two?